docs/user-guide/certificate-configuration-scenarios.md

```diff
@@ -290,6 +289,13 @@ this field is not defined, the `zwe init` command uses the value `zowe.externalD
      - "zOSMFCA"
 ```
 
+If you would like to use this example in your Zowe configuration YAML file, you just need to replace four fields with your own ones.
+
+1. Replace `ZoweKeyringZOSMF` with the your own key ring name.
+2. Replace `IZUSVR` with your user name.
```

Comment:
The `IZUSVR` user name is the user ID that owns the keyring you're trying to connect to, and not the user's current ID, so to avoid confusion maybe line 295 should be:
2. Replace 'IZUSVR' with the user name who is the owner of the existing certificate.